### PR TITLE
[ADD] Accounting: new page about intrastat

### PR DESCRIPTION
**Pull request by chiaraprattico**
Task-id 2721400

https://www.odoo.com/web#id=2721400&menu_id=4720&cids=1&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/finance/accounting/reporting/declarations/intrastat.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+   As of 1st January 2022, the reporting requirements have changed. One main modification concerns
+   the transactions’ codes, which used to be 1-digit format, but they are now switching to a 2-digit
+   one. Any transaction recorded before this date should be registered with the one-digit code.
+   Instead, all transactions after that date have to follow the new system’s rules.
+
+Configuration
+=============
+
+Go to :menuselection:`Accounting --> Settings --> Customer Invoices, then enable Intrastat and Save`
+Once the Intrastat has been activated you are able to choose the *Incoterm* values.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/intrastat.rst:20]:
```suggestion
Once the Intrastat has been activated you are able to choose the **Default incoterm**.
```

File: content/applications/finance/accounting/reporting/declarations/intrastat.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.. important::
+   If you created your Odoo database prior to **January 13, 2022**, please :ref:`upgrade
+   <general/upgrade>` ``account_intrastat`` to add the new transaction codes and :ref:`install
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/intrastat.rst:24]:
```suggestion
   <general/upgrade>` the ``account_intrastat`` module to add the new transaction codes and :ref:`install
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hi @chiaraprattico :)

Nice one for your first doc PR!

This page isn't covering everything about Intrastat, but we'll update it later. What matter now is the last section on this page.

Please re-request a review from me once you've updated your branch.

Cheers,
Jonathan

File: content/applications/finance/accounting/reporting/declarations/intrastat.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+provide data of the goods traded with other EU countries.
+
+.. seealso::
+   - `Intrastat
+     <https://ec.europa.eu/eurostat/statistics-explained/index.php?title=Glossary:Intrastat>`_
+
+.. note::
+   As of January 1, 2022, the reporting requirements have changed. One main modification concerns
+   the transactions codes which are now switching to a 2-digits format.
+   All transactions recorded after this date have to follow the new system’s rules.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/intrastat.rst:16]: Break the line at 100th character
```suggestion
   As of January 1, 2022, the reporting requirements have changed. One main modification concerns
   the transactions codes which are now switching to a 2-digits format. All transactions recorded
   after this date have to follow the new system’s rules.
```

File: content/applications/finance/accounting/reporting/declarations/intrastat.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Configuration
+=============
+
+Go to :menuselection:`Accounting --> Settings --> Customer Invoices`, then enable **Intrastat** and
+*Save*.```
+Once the Intrastat has been activated you are able to choose the **Default incoterm**.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/intrastat.rst:23]: - why the three ``` ?
- line break at 100th character

```suggestion
Go to :menuselection:`Accounting --> Settings --> Customer Invoices`, then enable **Intrastat** and
*Save*. Once the Intrastat has been activated you are able to choose the **Default incoterm**.
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Small changes

**Review by jcs-odoo — APPROVED** [review]
Good job for your first PR :)
I now can request a review from @odoo/doc-review . Either they merge it themselves or they'll delegate you the rights to do so and you'll be able to merge the PR yourself, including the fw-port PR as well.

File: content/applications/finance/accounting/reporting/declarations/intrastat.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+EU Member States. Organizations whose annual trade surpasses a certain threshold must provide data
+of the goods traded with other EU countries.
+
+.. seealso::
+   - `Eurostat Statistics Explained - Glossary: Intrastat
+     <https://ec.europa.eu/eurostat/statistics-explained/index.php?title=Glossary:Intrastat>`_
+
+.. note::
+   As of January 1, 2022, the reporting requirements have changed. One main modification concerns
+   the transactions codes which are now switching to a 2-digits format. All transactions recorded
```
Comment (Feyensv) [inline content/applications/finance/accounting/reporting/declarations/intrastat.rst:15]:
```suggestion
   the transaction codes which are now switching to a 2-digits format. All transactions recorded
```

File: content/applications/finance/accounting/reporting/declarations/intrastat.rst [hunk 6]
```diff
@@ -0,0 +1,28 @@
+=========
+Intrastat
+=========
+
+**Intrastat** is the system for data collection and statistic production of goods traded among the
+EU Member States. Organizations whose annual trade surpasses a certain threshold must provide data
+of the goods traded with other EU countries.
```
Comment (Feyensv) [inline content/applications/finance/accounting/reporting/declarations/intrastat.rst:7]:
```suggestion
about the goods traded with other EU countries.
```

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**